### DISCUSSION TITLE
Hey, No Biggie

### DISCUSSION THREAD

**Discussion by HarriFent** [bladewindui/ui#217] [General]
Hey I recently contributed to fixing 
https://github.com/mkocansey/bladewind/issues/206

And I noticed you attributed it to someone else on this release note.
https://github.com/mkocansey/bladewind/releases/tag/v2.5.2

It is no big deal but having contributions to my account will help with my career and stuff

**Comment by mkocansey** [comment]
My apologies .. @HarriFent .. that was completely an oversight.
I have updated the release notes, rightfully attributing the contribution to you. 
Thanks for bringing this up.

**Reply by HarriFent** [reply]
Thank you <3
